tests/lib/SimpleSAML/DatabaseTest.php

```diff
@@ -245,21 +245,22 @@ public function querying()
         $this->assertEquals($this->config->getString('database.prefix')."sspdbt", $table);
 
         $this->db->write(
-            "CREATE TABLE IF NOT EXISTS $table (ssp_key INT(16) NOT NULL, ssp_value TEXT NOT NULL)",
-            false
+            "CREATE TABLE IF NOT EXISTS $table (ssp_key INT(16) NOT NULL, ssp_value TEXT NOT NULL)"
         );
```

Comment:
An actual bug;  no second param is allowed

Reply:
On second view, a second param _is allowed_, it just cannot be false..
For 2.0 we can actually refactor `Database::write()` and leave out the `else`-case